enterprise_gateway/services/processproxies/container.py

```diff
@@ -197,21 +200,34 @@ async def confirm_remote_startup(self) -> None:
             await self.handle_timeout()
 
             container_status = self.get_container_status(i)
+            self.log.debug(
+                f">>> container.confirm_remote_startup() - container_status: {container_status}"
+            )
             if container_status:
                 if container_status in self.get_error_states():
                     self.log_and_raise(
                         http_status_code=500,
                         reason=f"Error starting kernel container; status: '{container_status}'.",
                     )
                 else:
+                    self.log.debug(
+                        f">>> container.confirm_remote_startup(): is hosted assigned => {self.assigned_host}"
+                    )
+                    self.log.debug(">>> should call receive_connection_info()")
                     if self.assigned_host:
                         ready_to_connect = await self.receive_connection_info()
+                        self.log.debug(
+                            f">>> container.confirm_remote_startup(): ready to connect => {ready_to_connect}"
```

Comment:
I'm cool with leaving these debug statements in - perhaps we can remove some of them later. However, can we avoid having two statements display the same message?  This statement is the same as that on L229, so it would be good differentiate these.